code/plotting.py

```diff
@@ -34,30 +34,30 @@ def draw_graph(thisGraph, graphTitle, MAIN_DIRECTORY, edgeWeight=None):
     '''
 
     imgFilename = MAIN_DIRECTORY + '/images/' + graphTitle + '.gif'
-    fig = plt.figure(facecolor='white', figsize=(16,12))
+    fig = plt.figure(facecolor='white', figsize=(16, 12))
 
-    edge95 = [(u,v) for (u,v,d) in thisGraph.edges(data=True) if d['weight'] == edgeWeight[0]]
-    edge90 = [(u,v) for (u,v,d) in thisGraph.edges(data=True) if d['weight'] == edgeWeight[1]]
-    edegeOverlap = [(u,v) for (u,v,d) in thisGraph.edges(data=True) if d['weight'] == edgeWeight[2]]
+    edge95 = [(u, v) for (u, v, d) in thisGraph.edges(data=True) if d['weight'] == edgeWeight[0]]
```

Comment:
for readability purposes, this is a general exception to the rule. 
[(u,v) for (u,v,d) in ...] is more readable than [(u, v) for (u, v, d) in ...]
like wise
fig = plt.figure(facecolor='white', figsize=(16,12))
instead of fig = plt.figure(facecolor='white', figsize=(16, 12))